torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -773,6 +773,12 @@ Tensor trace_backward(const Tensor & grad, IntArrayRef sizes) {
   return grad_input.view(sizes);
 }
 
+Tensor sgn_backward(Tensor result, Tensor grad, Tensor self) {
+  // [grad / abs(z) - Re(grad/self) * result
```

Comment:
I think there are mismatched parenthesis here no?
Also `z` is not defined.
Also the special case at 0 might need to be mentioned if you explain what this function is computing.